src/azure-cli/azure/cli/command_modules/resource/_params.py

```diff
@@ -294,7 +295,9 @@ def load_arguments(self, _):
         c.argument('include_parameter_default_value', action='store_true')
 
     with self.argument_context('group create') as c:
-        c.argument('rg_name', options_list=['--name', '--resource-group', '-n', '-g'], help='name of the new resource group', completer=None)
+        c.argument('rg_name', options_list=['--name', '--resource-group', '-n', '-g'],
+                   help='name of the new resource group', completer=None, lc_actions=[STORE], lc_scopes=[GLOBAL],
+                   lc_name='resource_group_name')
```

Comment:
is `lc_name` arbitrary? I mean, can developer choose any name? It's just the key to store the value. Is it correct?

Reply:
yes, developer should make sure that lc_name used in STORE places are consistent with USE places.